Lesson32/Ezgi/my-app/src/shared-components/typography/BodyS/styles.css

```diff
@@ -0,0 +1,9 @@
+.body-s {
```

Comment:
This will not work since 'body-s' class is not added anywhere in the BodyS component. Instead, use  'italic' class that you have added. 
Since the only difference between BodyM and BodyS are the font-style, feel free to only add 'font-style: italic;' here.

Reply:
I'd been forgot to change the class name. I've updated it to italic. Thanks :)

Reply:
No worries :)